src/main/java/hudson/plugins/git/extensions/impl/ChangelogToBranch.java

```diff
@@ -25,6 +26,15 @@ public ChangelogToBranch(ChangelogToBranchOptions options) {
         this.options = options;
     }
 
+    @Override
+    public boolean requiresWorkspaceForPolling() {
+	// Documentation in help.html (probably copy/pasted) claimed that this extension required
+	// a workspace for polling.  However, in fact it did not prior to work on JENKINS-51633.
+	// Rather than always return true, maintain backward compatability since isLocalTarget() will
+	// be false for all previously valid configurations.
+        return options.isLocalTarget();
+    }
```

Comment:
Yea, so we just require workspace for polling, meaning that we'll never trigger code for if we're doing remote polling..?